Casks/safenet-authentication-client.rb

```diff
@@ -7,6 +7,10 @@
   desc "Client for smart card readers and security tokens"
   homepage "https://support.globalsign.com/ssl/ssl-certificates-installation/safenet-drivers"
 
+  livecheck do
+    skip "No version information available"
+  end
+
   depends_on macos: ">= :catalina"
```

Comment:
This cask in its current format will only run on Catalina and not Big Sur.
It looks like a fairly major re-write to incorporate the Big Sur version, as it has a different install / uninstall procedure.

Thoughts @miccal ?

```suggestion
  depends_on macos: :catalina
```

Reply:
Let's set this for now, and if a user wants to add the Big Sur version they can submit a PR or open an issue.